ShopifySharp.Tests/ShippingZone_Tests.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ShopifySharp.Tests
+{
+    [Trait("Category", "ShippingZone")]
+    public class ShippingZone_Tests
+    {
+        private ShippingZoneService _Service => new ShippingZoneService(Utils.MyShopifyUrl, Utils.AccessToken);
+
+        [Fact]
+        public async Task Lists_ShippingZones()
+        {
+            var list = await _Service.ListAsync();
+
+            Assert.NotNull(list);
```

Comment:
Could you please expand the test fixture to validate the properties of any shipping zone that was returned in the list? For example, validate that for every item the `Name` is not null, etc.